### PR TITLE
Update version and packages- SpotAPI

### PR DESCRIPTION
**Pull request by nvnyale**

### CODE REVIEW

**Comment by tenyo** [comment]
The CircleCI test job is failing. We've actually started switching to Github Actions from CircleCI in other places. Can you remove the .circleci config from this repo and replace it with .github (you should be able to copy https://github.com/YaleSpinup/ec2-api/blob/master/.github/workflows/test.yaml)

File: go.mod [hunk 1]
```diff
@@ -1,22 +1,32 @@
...
-	github.com/golang/protobuf v1.5.2 // indirect
-	github.com/gopherjs/gopherjs v0.0.0-20210202160940-bed99a852dfe // indirect
 	github.com/gorilla/handlers v1.5.1
 	github.com/gorilla/mux v1.8.0
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.10.0
-	github.com/prometheus/common v0.24.0 // indirect
 	github.com/sirupsen/logrus v1.8.1
-	github.com/smartystreets/assertions v1.2.0 // indirect
 	github.com/spotinst/spotinst-sdk-go v1.87.0
```
Comment (tenyo) [inline go.mod]: hmm, there should be a newer version of this package
did you run `go get -u`?
also run `go mod tidy`

**Review by tenyo — CHANGES_REQUESTED** [review]

**Comment by tenyo** [comment]
Looks good, let's make a pre-release

File: go.mod (hunk 1 above)
Comment (nvnyale) [reply go.mod]: done.

**Pull request merged**